src/explore-education-statistics-frontend/src/components/__tests__/Feedback.test.tsx

```diff
@@ -0,0 +1,133 @@
+import Feedback from '@frontend/components/Feedback';
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import _feedbackService from '@frontend/services/feedbackService';
+import { FeedbackRequest } from '@common/services/types/feedback';
+
+jest.mock('@frontend/services/feedbackService');
+
+const feedbackService = jest.mocked(_feedbackService);
+
+describe('Feedback', () => {
+  test('renders initial state correctly', () => {
+    render(<Feedback />);
+    expect(screen.getByText('Is this page useful?')).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Yes this page is useful' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'No this page is not useful' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Report a problem with this page' }),
+    ).toBeInTheDocument();
+  });
+
+  test('renders correctly and submits when "Yes" is selected', async () => {
+    render(<Feedback />);
+    await userEvent.click(
+      screen.getByRole('button', { name: 'Yes this page is useful' }),
```

Comment:
We should check that the text isn't visible before the button is pressed as well to check that a change actually occurred